### ISSUE TITLE
Avatars missing/mis-configured?

### ISSUE THREAD

**Issue by tensionhead** [haiwen/seafile-rpi#27]
Hello,

during initial web configuration of the latest server version for the pi (downloaded today) I get the following error:

`2017/12/16 15:08:46 [error] 2158#0: *22 open() "/home/xxx/sfile/seafile-server-latest/seahub/media/avatars/default.png" failed (2: No such file or directory), client: 192.168.178.100, server: my.server.com, request: "GET /seafmedia/avatars/default.png HTTP/1.1", host: "rasp2", referrer: "https://rasp2/sea/profile/"`

And indeed, if I peek in to the 

`/seafile-server-latest/seahub/`

directory, I see a broken link to 

`avatars -> ../../../seahub-data/avatars`

where in this location /seahub-data/avatars there is just the same link set again.

Consequently some images in the web interface are broken.

**Comment by dariuspov** [comment]
Hi, I've had same issue. After defauld Seafile deployment - on user avatar I was geting 
![image](https://user-images.githubusercontent.com/11914760/38990325-80073ef2-43e2-11e8-807e-97a02036bb94.png)

Then when I tried to upload image via web browser it returned an error.

So I've found the directory on seahub-data/ and found broken link as example below
![image](https://user-images.githubusercontent.com/11914760/38990292-45a61198-43e2-11e8-9c0f-a638068931ae.png)

What I did - is removed this avatar file, created new folder avatar, gave all needed permissions (user and read/write) and this helped. Now I am able to change avatar on user settings and avatar folder now has a file. 

Hope that helps,
